Sources/SwiftSentry/SentrySDK.swift

```diff
@@ -203,10 +203,13 @@ public enum SentrySDK {
 
         // Log the outer crash stack trace and all the stowed exceptions as distinct exception events.
         // The events will be displayed in the Sentry UI as a single event with multiple stack traces.
-        let exceptions = sentry_value_new_list();
+        let exceptions = sentry_value_new_list()
         let exception = sentry_value_new_exception("Outer crash", "Outer crash with stowed exceptions")
+
+        setTag(key: "handled", value: "no")
```

Comment:
Do you need a "handled" tag? I thought the `sentry_value_set_by_key(mechanism, "handled", sentry_value_new_bool(Int32(false)))` would be enough?

Reply:
It's enough, but I think that adding the handled tag is more consistent with the other crashes that we have

Reply:
Oh, I thought that adding `handled` to the mechanism would make it show up in "tags" automatically, apparently not. No problem, just a nit.